Makefile

```diff
@@ -10,4 +11,6 @@ deploy:
 	cd api && NODE_ENV=production make deploy
 
 publish:
+	cd admin && make build
+	mv admin/build cli/admin
```

Comment:
This will significantly increase the cli size.
I would suggest that the admin should be published in a separated lib `comfygure-admin` and that the command `comfy admin` check weither the package is installed or not.

The installation instruction would be:
```bash
npm install -g comfygure-admin
comfy admin
```